test/images/Makefile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REGISTRY ?= gcr.io/google_containers
```

Comment:
@ixdy So this isn't really adding "new" image per se, but just keeping backwards compat with everything that is there now.

Personally I'd be delighted if we could move everything in this dir to something but `k8s-testimages` as well, maybe `k8s-e2e-images` or something. As you said `k8s-testimages` is already being used for test infra stuff...